<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>431</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Категория:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="boxCategory">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Заголовок:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtTitle">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btnCopy">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/copyIcon.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnCut">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/cut-fr.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnPaste">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/pasteIcon.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnUndo">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/undoIcon.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRedo">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/redoIcon.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnFont">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="bin/img.qrc">
         <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="edtText">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/add1.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="bin/img.qrc">
     <normaloff>:/images/copyIcon.png</normaloff>:/images/copyIcon.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="bin/img.qrc">
     <normaloff>:/images/cut-fr.png</normaloff>:/images/cut-fr.png</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="bin/img.qrc">
     <normaloff>:/images/pasteIcon.png</normaloff>:/images/pasteIcon.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="bin/img.qrc">
     <normaloff>:/images/undoIcon.png</normaloff>:/images/undoIcon.png</iconset>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="bin/img.qrc">
     <normaloff>:/images/redoIcon.png</normaloff>:/images/redoIcon.png</iconset>
   </property>
   <property name="text">
    <string>Вперёд</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="bin/img.qrc">
     <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
   </property>
   <property name="text">
    <string>Шрифт</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>edtText</tabstop>
  <tabstop>btnRedo</tabstop>
  <tabstop>btnFont</tabstop>
  <tabstop>btnUndo</tabstop>
  <tabstop>btnPaste</tabstop>
  <tabstop>edtTitle</tabstop>
  <tabstop>btnCut</tabstop>
  <tabstop>boxCategory</tabstop>
  <tabstop>btnCopy</tabstop>
 </tabstops>
 <resources>
  <include location="bin/img.qrc"/>
 </resources>
 <connections/>
</ui>
